go-controller/pkg/ovn/common.go

```diff
@@ -21,6 +21,32 @@ func hashedAddressSet(s string) string {
 	return fmt.Sprintf("a%s", hashString)
 }
 
+func (oc *Controller) forEachAddressSet(iteratorFn func(string, string,
+	string)) error {
+	// get the list of address sets from OVN
+	output, err := exec.Command(OvnNbctl, "--data=bare", "--no-heading",
+		"--columns=external_ids", "find", "address_set").Output()
+	if err != nil {
+		logrus.Errorf("Error in obtaining list of address sets from OVN: %v",
+			err)
+		return err
+	}
+	for _, addrSet := range strings.Fields(string(output)) {
+		if !strings.HasPrefix(addrSet, "name=") {
+			continue
+		}
+		addrSetName := addrSet[5:]
+		names := strings.Split(addrSetName, ".")
+		asNamespace := names[0]
```

Comment:
Can you please rename asNamespace as something like "addrsSetNamespace"? People may read it as "as namespace".

Reply:
Done